release_tester/arangodb/starter/deployments/dc2dc.py

```diff
@@ -208,6 +208,29 @@ def _is_higher_sync_version(self, min_v1_version, min_v2_version):
 
         return is_higher_version(self.sync_version, min_v2_version)
 
+    def _get_sync_status(self, target_cluster=True):
+        """
+            Check replication status of the cluster.
+        """
+        cluster_instance = self.cluster1['instance']
```

Comment:
please use 
```
cluster = self.cluster1 if not target_cluster else self.cluster
```
and work with sub-references of cluster below.

Reply:
changed